SPECS/cal10n/cal10n.spec

```
@@ -73,58 +74,54 @@ http://cal10n.qos.ch/manual.html
 EOF
 
 %build
-for dir in cal10n-api
-do
-  pushd $dir
-  export CLASSPATH=$(build-classpath \
-                     junit \
-                     ):target/classes:target/test-classes
-  ant -Dmaven.mode.offline=true package javadoc \
-      -Dmaven.test.skip=true \
-      -lib %{_datadir}/java
-  popd
-done
+mkdir -p lib
+build-jar-repository -s lib \
+%if %{with_check}
+    ant-antunit \
+%endif
+    ant/ant
+%{ant} \
+%if %{without tests}
+    -Dtest.skip=true \
+%endif
+    package javadoc
```

We're closer but we still have `without tests` and want to use `?`:
```suggestion
%if 0%{?with_check}
    ant-antunit \
%endif
    ant/ant
%{ant} \
%if ! 0%{?with_check}
    -Dtest.skip=true \
%endif
    package javadoc
```